src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -136,8 +136,14 @@ class DatabaseFeedManagerMessageProcessor final
                 }
 
                 std::string line;
+                int32_t step = 0;
                 while (std::getline(file, line))
                 {
+                    if (step++ % 100 == 0)
+                    {
+                        logInfo(WM_VULNSCAN_LOGTAG, "Processing line: %d", step);
```

Comment:
I'd simply add this log for all lines but in debug.

Reply:
Done